.gitignore

```diff
@@ -5,6 +5,8 @@
 ## Ignore Visual Studio temporary files, build results, and
 ## files generated by popular Visual Studio add-ons.
 
+.nuke/temp
```

Comment:
This is here because at some point Nuke had crashed while building, leaving this temp folder. I had accidentally committed it locally without noticing, so I added it here as a catch for myself. I can remove it.